app/views/application/_multi_answer_row.pdf.erb

```diff
@@ -0,0 +1,19 @@
+<tr>
+  <td class="question">
+    <%= translate_with_appeal_or_application "check_answers.#{multi_answer_row.question}.question" %>
+  </td>
+
+  <td class="answer answer-multi-choice">
+    <ul class="list list-bullet">
+      <% multi_answer_row.value.each do |value| %>
+        <% if value.is_a?(String) && value.present? %>
```

Comment:
same as above

Reply:
I've moved the type checking to a helper method

Reply:
cool thx.